App/views.py

```diff
@@ -0,0 +1,66 @@
+import json
+import requests
+import urllib.parse
+
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.utils.http import urlquote
+
+
+def get_article_view_count(request):
+    if request.method=='POST':
+        username = request.POST['username']
+        try:
+            if not username:
+                raise Error("Please enter a username")
+            params = {'action':'query',
+                      'format':'json',
+                      'list':'usercontribs',
+                      'uclimit':'5',
+                      'ucuser':username }
+            url_base = 'https://en.wikipedia.org/w/api.php?'
+            # Recieved data in json-format
+            response = requests.get(url_base, params = params)
+            if response.status_code == 200:
```

Comment:
You can use `response.raise_for_status()` and let exception handling deal with non-200 responses.